Example Throws TypeError

With a fresh install of `cmd2` and `cmd2-ext-test`, the example from the README will generate a type error. To recreate:

``` bash
mkdir cmd2_test && cd cmd2_test
python3 -m venv venv
. venv/bin/activate
pip3 install cmd2 cmd2-ext-test
```
Copy all code form the README into a file, test.py:
```python
import cmd2
import cmd2_ext_test
import pytest

from cmd2 import CommandResult


class ExampleApp(cmd2.Cmd):
    """An class to show how to use a plugin"""
    def __init__(self, *args, **kwargs):
        # gotta have this or neither the plugin or cmd2 will initialize
        super().__init__(*args, **kwargs)

    def do_something(self, arg):
        self.last_result = 5
        self.poutput('this is the something command')


class ExampleAppTester(cmd2_ext_test.ExternalTestMixin, ExampleApp):
    def __init__(self, *args, **kwargs):
        # gotta have this or neither the plugin or cmd2 will initialize
        super().__init__(*args, **kwargs)


@pytest.fixture
def example_app():
    app = ExampleAppTester()
    app.fixture_setup()
    yield app
    app.fixture_teardown()


def test_something(example_app):
    # execute a command
    out = example_app.app_cmd("something")

    # validate the command output and result data
    assert isinstance(out, CommandResult)
    assert str(out.stdout).strip() == 'this is the something command'
    assert out.data == 5
```
And then run pytest:

```bash
$ pytest test.py
============================================================================================ test session starts =============================================================================================
platform linux -- Python 3.8.2, pytest-6.0.1, py-1.9.0, pluggy-0.13.1
rootdir: /home/charlie/cmd2_test
plugins: cov-2.10.0
collected 1 item                                                                                                                                                                                             

test.py F                                                                                                                                                                                              [100%]

================================================================================================== FAILURES ==================================================================================================
_______________________________________________________________________________________________ test_something _______________________________________________________________________________________________

example_app = <test.ExampleAppTester object at 0x7f67e881ed90>

    def test_something(example_app):
        # execute a command
>       out = example_app.app_cmd("something")

test.py:35: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <test.ExampleAppTester object at 0x7f67e881ed90>, command = 'something', echo = None

    def app_cmd(self, command: str, echo: Optional[bool] = None) -> cmd2.CommandResult:
        """
        Run the application command
        :param command: The application command as it would be written on the cmd2 application prompt
        :param echo: Flag whether the command's output should be echoed to stdout/stderr
        :return: A CommandResult object that captures stdout, stderr, and the command's result object
        """
        try:
            self._in_py = True
    
>           return self._pybridge(command, echo)
E           TypeError: __call__() takes 2 positional arguments but 3 were given

../.local/lib/python3.8/site-packages/cmd2_ext_test/cmd2_ext_test.py:29: TypeError
========================================================================================== short test summary info ===========================================================================================
FAILED test.py::test_something - TypeError: __call__() takes 2 positional arguments but 3 were given
============================================================================================= 1 failed in 0.27s ==============================================================================================
```
Output of `pip3 freeze`:
```bash
$ pip3 freeze
attrs==19.3.0
cmd2==1.2.1
cmd2-ext-test==0.1.0
colorama==0.4.3
pyperclip==1.8.0
wcwidth==0.2.5
```

--- a/cmd2_ext_test/cmd2_ext_test.py
+++ b/cmd2_ext_test/cmd2_ext_test.py
@@ -26,7 +26,7 @@ class ExternalTestMixin:
         try:
             self._in_py = True
 
-            return self._pybridge(command, echo)
+            return self._pybridge(command, echo=echo)
 
         finally:
             self._in_py = False

